tensorflow/tools/dockerfiles/partials/mpi.partial.Dockerfile

```diff
@@ -0,0 +1,44 @@
+# install libnuma, openssh, wget
+RUN apt-get update && apt-get install -y --no-install-recommends --fix-missing \
+        libnuma-dev \
+        openssh-server \
+        openssh-client \
+        wget && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* || \
+    yum -y update && yum -y install \
```

Comment:
If this is under Ubuntu, is `yum` going to be available?

Reply:
This partial is designed to work with both Ubuntu and CentOS. So, the RUN command will work on either base OS. That is why it is not under the Ubuntu folder.